ckanext/cfpb_extrafields/templates/package/snippets/package_basic_fields.html

```diff
@@ -2,12 +2,16 @@
 
 
 {% block package_basic_fields_tags %} {# unchanged#}
+<div id="core-field">
```

Comment:
id's should be unique.  In this case, I think you may want to use a class instead.